pkg/backup/backup.go

```diff
@@ -189,6 +189,9 @@ func (c *Controller) setup() (*api.Restic, error) {
 }
 
 func (c *Controller) runResticBackup(resource *api.Restic) (err error) {
+	if resource.Spec.Paused == true {
+		return nil
```

Comment:
log.InfoLn("skipped logging since restic is paused.")

Reply:
added.